openfecwebapp/__init__.py

```diff
@@ -0,0 +1,191 @@
+import re
+import json
+import locale
+import hashlib
+import datetime
+
+import furl
+from raven.contrib.flask import Sentry
+
+from hmac_authentication import hmacauth
+from flask import Flask, render_template, request, url_for
+from flask_sslify import SSLify
+from werkzeug.contrib.fixers import ProxyFix
+
+from openfecwebapp import utils
+from openfecwebapp import views
+from openfecwebapp import config
+from openfecwebapp.config import env
+from openfecwebapp import constants
+
+
+locale.setlocale(locale.LC_ALL, '')
+
+app = Flask(__name__, static_path='/static', static_folder='../dist')
+
+from openfecwebapp import routes  # noqa
+from openfecwebapp import filters  # noqa
+
+
+def get_absolute_url():
+    url = furl.furl(request.url)
+    if app.config['SERVER_NAME']:
+        url.host = app.config['SERVER_NAME']
+        url.scheme = app.config['PREFERRED_URL_SCHEME']
+    return url.url
+
+
+def _get_default_cycles():
+    cycle = utils.current_cycle()
+    return list(range(cycle - 4, cycle + 2, 2))
+
+
+def series_has_data(values, key):
+    return next(
+        (True for value in values if value.get(key) is not None),
+        False,
+    )
```

Comment:
Honestly, I don't remember--this is all old code that would ideally be torn out and replaced. Feel free @ccostino!